src/routes/admin/index.ts

```
@@ -1,10 +1,13 @@
 import { Router } from 'express';
 import authRoutes from './auth.routes.js';
+import adminsRoutes from './admins.routes.js';
 import analyticsRoutes from './analytics.routes.js';
 import merchantRoutes from './merchant.routes.js';
 import logsRoutes from './logs.routes.js';
 import subscriptionsRoutes from './subscriptions.routes.js';
+import subscriptionPlansRoutes from './subscription-plans.routes.js';
 import invoiceRoutes from './invoice.routes.js';
+import subscriptionPlansRoutes from './subscription-plans.routes.js';
```

ESLint

[error] './subscription-plans.routes.js' import is duplicated.

(no-duplicate-imports)